packages/flutter/test/rendering/paragraph_test.dart

```diff
@@ -271,17 +271,14 @@ void main() {
     // rendering widths in tests.
     // TODO(#12357): Figure out why this is, and fix it (if needed) once Blink
     // text rendering is replaced.
-    // anyOf for heights is needed because libtxt and Blink calculate selection
-    // rectangles differently.
-    // TODO: remove this when Blink is replaced.
     expect(boxes[0].toRect().width, anyOf(14.0, 13.0));
-    expect(boxes[0].toRect().height, anyOf(13.0, 26.0));
+    expect(boxes[0].toRect().height, closeTo(13.0, 0.0001));
```

Comment:
btw, you can use moreOrLessEquals instead of closeTo if you don't want to specify the 0.0001 explicitly (up to you)